### PR TITLE
Added jest

### PR DESCRIPTION
**Pull request by xandora**
Removed generator-exercises folder as it breaks jest-codemods

run jest-codemods on .spec.js files, move generator-exercises back in

Change references from Jasmine to Jest in main readme

Update README with Jest specific language. Update some spec files with new syntax

update tests, multiple exercises

.gitignore: Added package-lock.json, package.json that were used when I ran code-blocks over the tests.

Standardised function declaration calls across exercises

fix typo in caesar.spec.js

Ignoring package-lock.json, package.json

Backtrack on .gitignore modification, add instructions to readme

move files from testing repo to this repo

Typo fixes, remove duplicate exercise folder

Remove solution from non-solution branch

Minor grammatical fixes

added trailing semicolon to all function and module exports

Fix words caught by search/replace action.

remove doubled semicolon.

Correct words caught by search/replace action.

Add missing semicolon.

Add .DS_Store to .gitignore

multiple files: Added a blank line at the end of each file

Ignore generator-exercise when linting exercise files

Update exercise number of each exercise

Update exercise number

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1 @@
+generator-exercise/
```
Comment (xandora) [inline .eslintignore:1]:
```suggestion
generator-exercise/

```

File: .gitignore [hunk 2]
```diff
@@ -1 +1,3 @@
-.vscode
\ No newline at end of file
+.vscode
+node_modules/
+.DS_Store
```
Comment (xandora) [inline .gitignore:3]:
```suggestion
.DS_Store

```

File: calculator/calculator.js [hunk 3]
```diff
@@ -1,32 +1,32 @@
...
-}
+};
 
-function multiply () {
-	
-}
+const multiply = function() {
 
-function power() {
+};;
```
Comment (xandora) [inline calculator/calculator.js:15]:
```suggestion
};
```

File: helloWorld/helloWorld.js [hunk 4]
```diff
@@ -2,4 +2,4 @@ const helloWorld = function() {
   return ''
 }
```
Comment (xandora) [inline helloWorld/helloWorld.js:3]:
```suggestion
};
```

File: package.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    "eslint": "^7.26.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.22.1"
+  },
+  "scripts": {
+    "test": "jest"
+  },
+  "eslintConfig": {
+    "root": true
+  }
```
Comment (xandora) [inline package.json:28]:
```suggestion
  },
   "jest": {
    "testPathIgnorePatterns": [
      "generator-exercise/"
    ]
  }
```

**Review by KevinMulhern — APPROVED** [review]
Looks great to me @xandora, thanks for all your work on this 💪

**Pull request merged**